lib/messages.rb

```diff
@@ -0,0 +1,21 @@
+class Messages
+  require 'twilio-ruby'
+  SID = ENV["SID"]
+  TOKEN = ENV["TOKEN"]
+  TWI = ENV["TWILIO_NUMBER"]
+  TO = ENV["PHONE"]
+
+  def initialize(from = TWI, to = TO, account_sid = SID, auth_token = TOKEN)
+    @client = Twilio::REST::Client.new account_sid, auth_token
+    @from =  from
+    @to = to
+    current = Time.now
+    @t = format('%d:%02d', current.hour + 1, current.min)
```

Comment:
Naming - don't use single letter abbreviations - you will forget what they stand for.